### PR TITLE
Fix cross icon bug

### PR DESCRIPTION
**Pull request by bingxie**

### CODE REVIEW

**Comment by rap1ds** [comment]
Hi @bingxie 

Thanks for the fix!

What's the `.icon-remove` class? What does it do?

**Comment by bingxie** [comment]
.icon-remove is the cross icon class from font-awesome.  This bit of css align the font-awesome cross icon in the center.

File: app/assets/stylesheets/main.css.scss [hunk 1]
```diff
@@ -283,6 +283,11 @@ ul {
   padding-top: lines(0.05);
 }
 
+.icon-remove {
+  top: -2px !important;
```
Comment (Gnito) [inline app/assets/stylesheets/main.css.scss]: Does this work without "!important". I would go to great lengths to avoid using that one.

Comment (bingxie) [inline app/assets/stylesheets/main.css.scss]: Good idea, I updated it. @Gnito Can you have a look again?

**Comment by bingxie** [comment]
I have updated the CSS, please have a look again.

**Comment by mporkola** [comment]
Hi @bingxie, could you rebase this to the current master? Sorry for the delay.

**Comment by bingxie** [comment]
I have rebased it on master.

**Comment by rap1ds** [comment]
Inactive.

**Pull request closed without merging**